geotrek/trekking/parsers.py

```diff
@@ -957,6 +957,8 @@ def _maybe_get_linestring_from_layer(layer):
         geos = ApidaeTrekParser._convert_to_geos(first_feature.geom)
         if geos.geom_type == 'MultiLineString':
             geos = geos.merged
+            if geos.geom_type == 'MultiLineString':
+                raise RowImportError("The geometry cannot be converted to a single continuous LineString feature")
```

Comment:
This may be translated here ? (not sure if error messages are translated)

Reply:
`RowImportErrors` show up in the import report so they should indeed be translated. Done.